INFO 2023-09-29 02:27:44,052 autoreload 6197 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 02:27:48,349 basehttp 6197 6126907392 "GET / HTTP/1.1" 200 13
INFO 2023-09-29 02:29:27,978 autoreload 6197 8539431040 /Users/aleksey/Desktop/Django/Seminar/myproject/settings.py changed, reloading.
INFO 2023-09-29 02:29:28,342 autoreload 6322 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 02:32:26,651 autoreload 6322 8539431040 /Users/aleksey/Desktop/Django/Seminar/myproject/settings.py changed, reloading.
INFO 2023-09-29 02:32:26,993 autoreload 6506 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 02:33:43,399 autoreload 6506 8539431040 /Users/aleksey/Desktop/Django/Seminar/games_app/views.py changed, reloading.
INFO 2023-09-29 02:33:43,756 autoreload 6597 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 02:35:19,372 autoreload 6597 8539431040 /Users/aleksey/Desktop/Django/Seminar/games_app/views.py changed, reloading.
INFO 2023-09-29 02:35:19,734 autoreload 6714 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 02:39:25,364 autoreload 6714 8539431040 /Users/aleksey/Desktop/Django/Seminar/myproject/settings.py changed, reloading.
INFO 2023-09-29 02:39:25,720 autoreload 6988 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 05:32:03,198 autoreload 16380 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 05:32:20,257 autoreload 16406 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 05:32:22,841 basehttp 16406 6165524480 "GET / HTTP/1.1" 200 13
INFO 2023-09-29 05:32:34,428 views 16406 6165524480 Result: Сторона: reshka, время: 2023-09-29 05:32:34.423478+00:00
INFO 2023-09-29 05:32:34,432 basehttp 16406 6165524480 "GET /eagle/ HTTP/1.1" 200 68
INFO 2023-09-29 05:32:44,189 views 16406 6165524480 Result: Сторона: reshka, время: 2023-09-29 05:32:44.187546+00:00
INFO 2023-09-29 05:32:44,190 basehttp 16406 6165524480 "GET /eagle/ HTTP/1.1" 200 68
INFO 2023-09-29 05:32:47,406 views 16406 6165524480 Result: Сторона: reshka, время: 2023-09-29 05:32:47.404387+00:00
INFO 2023-09-29 05:32:47,407 basehttp 16406 6165524480 "GET /eagle/ HTTP/1.1" 200 68
INFO 2023-09-29 05:33:49,113 autoreload 16406 8539431040 /Users/aleksey/Desktop/Django/Seminar/games_app/views.py changed, reloading.
INFO 2023-09-29 05:33:49,459 autoreload 16541 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 05:33:58,728 autoreload 16541 8539431040 /Users/aleksey/Desktop/Django/Seminar/games_app/views.py changed, reloading.
INFO 2023-09-29 05:33:59,040 autoreload 16561 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 05:34:02,712 views 16561 6191394816 Result: Сторона: reshka, время: 2023-09-29 05:34:02.709348+00:00
INFO 2023-09-29 05:34:02,713 basehttp 16561 6191394816 "GET /eagle/ HTTP/1.1" 200 68
INFO 2023-09-29 05:34:05,818 views 16561 6191394816 Result: Сторона: orel, время: 2023-09-29 05:34:05.815302+00:00
INFO 2023-09-29 05:34:05,819 basehttp 16561 6191394816 "GET /eagle/ HTTP/1.1" 200 66
INFO 2023-09-29 06:18:09,666 autoreload 19845 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 06:19:20,656 autoreload 19845 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-09-29 06:19:20,962 autoreload 19925 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 06:19:33,147 autoreload 19925 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-09-29 06:19:33,393 autoreload 19951 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 06:19:55,369 autoreload 19976 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 06:20:13,164 autoreload 20004 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 06:20:15,188 basehttp 20004 6201470976 "GET / HTTP/1.1" 200 13
INFO 2023-09-29 06:20:19,069 basehttp 20004 6201470976 "GET / HTTP/1.1" 200 13
WARNING 2023-09-29 06:20:25,221 log 20004 6201470976 Not Found: /show_elements
WARNING 2023-09-29 06:20:25,222 basehttp 20004 6201470976 "GET /show_elements HTTP/1.1" 404 2992
INFO 2023-09-29 06:20:54,061 basehttp 20004 6201470976 "GET / HTTP/1.1" 200 13
WARNING 2023-09-29 06:20:54,687 log 20004 6201470976 Not Found: /show_elements
WARNING 2023-09-29 06:20:54,688 basehttp 20004 6201470976 "GET /show_elements HTTP/1.1" 404 2992
INFO 2023-09-29 06:21:26,948 basehttp 20004 6201470976 "GET /show_elements/3 HTTP/1.1" 301 0
INFO 2023-09-29 06:21:26,962 basehttp 20004 6201470976 "GET /show_elements/3/ HTTP/1.1" 200 232
INFO 2023-09-29 06:23:09,396 autoreload 20004 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-09-29 06:23:09,677 autoreload 20204 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 06:23:44,328 autoreload 20204 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-09-29 06:23:44,661 autoreload 20244 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 06:26:31,326 basehttp 20244 6200799232 "GET / HTTP/1.1" 200 13
INFO 2023-09-29 06:26:31,604 autoreload 20244 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-09-29 06:26:31,885 autoreload 20429 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 06:26:39,186 basehttp 20429 6171570176 "GET /show_elements/3/ HTTP/1.1" 200 214
INFO 2023-09-29 06:27:55,493 autoreload 20429 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-09-29 06:27:55,772 autoreload 20520 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 06:28:01,960 basehttp 20520 6137294848 "GET /show_elements/3/ HTTP/1.1" 200 226
INFO 2023-09-29 06:29:17,395 autoreload 20520 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-09-29 06:29:17,688 autoreload 20628 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 06:29:23,563 basehttp 20628 6155726848 "GET /show_elements/3/ HTTP/1.1" 200 226
INFO 2023-09-29 06:32:00,021 basehttp 20628 6155726848 "GET /show_elements/3/ HTTP/1.1" 200 226
INFO 2023-09-29 06:36:04,076 autoreload 20628 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-09-29 06:36:04,377 autoreload 21029 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 06:36:40,011 autoreload 21029 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-09-29 06:36:40,291 autoreload 21083 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 06:36:48,597 basehttp 21083 6132297728 "GET /show_elements/3/ HTTP/1.1" 200 412
INFO 2023-09-29 06:37:22,317 autoreload 21083 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-09-29 06:37:22,674 autoreload 21130 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 06:37:26,278 basehttp 21130 6166245376 "GET /show_elements/3/ HTTP/1.1" 200 403
INFO 2023-09-29 06:37:59,534 autoreload 21130 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-09-29 06:37:59,841 autoreload 21170 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 06:38:08,570 basehttp 21170 6203273216 "GET /show_elements/3/ HTTP/1.1" 200 404
INFO 2023-09-29 06:47:48,648 autoreload 21170 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-09-29 06:47:48,935 autoreload 21795 8539431040 Watching for file changes with StatReloader
ERROR 2023-09-29 06:47:56,611 log 21795 6168129536 Internal Server Error: /show_elements/3/
Traceback (most recent call last):
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/games_app/views.py", line 23, in show_elements
    return HttpResponse(f'{Coin.counter(n)}')
                           ^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/myapp2/models.py", line 16, in counter
    coins = Coin.objects.all()[-n:]
            ~~~~~~~~~~~~~~~~~~^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/db/models/query.py", line 429, in __getitem__
    raise ValueError("Negative indexing is not supported.")
ValueError: Negative indexing is not supported.
ERROR 2023-09-29 06:47:56,613 basehttp 21795 6168129536 "GET /show_elements/3/ HTTP/1.1" 500 66875
INFO 2023-09-29 06:49:18,695 autoreload 21795 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-09-29 06:49:18,968 autoreload 21898 8539431040 Watching for file changes with StatReloader
ERROR 2023-09-29 06:49:26,002 log 21898 6134689792 Internal Server Error: /show_elements/3/
Traceback (most recent call last):
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/games_app/views.py", line 23, in show_elements
    return HttpResponse(f'{Coin.counter(n)}')
                           ^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/myapp2/models.py", line 19, in counter
    if coin.is_eagle == 'orel':
       ^^^^^^^^^^^^^
AttributeError: 'QuerySet' object has no attribute 'is_eagle'
ERROR 2023-09-29 06:49:26,003 basehttp 21898 6134689792 "GET /show_elements/3/ HTTP/1.1" 500 64030
INFO 2023-09-29 06:50:14,011 autoreload 21898 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-09-29 06:50:14,334 autoreload 21964 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 06:51:15,577 autoreload 21964 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-09-29 06:51:15,930 autoreload 22038 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 06:51:19,145 basehttp 22038 6197555200 "GET /show_elements/3/ HTTP/1.1" 200 24
INFO 2023-09-29 06:53:50,854 autoreload 22038 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-09-29 06:53:51,127 autoreload 22206 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 06:53:57,206 basehttp 22206 6159396864 "GET /show_elements/3/ HTTP/1.1" 200 24
INFO 2023-09-29 07:00:19,190 autoreload 22206 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-09-29 07:00:19,523 autoreload 22607 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 07:02:14,631 autoreload 22607 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-09-29 07:02:14,921 autoreload 22737 8539431040 Watching for file changes with StatReloader
INFO 2023-09-29 07:02:18,087 autoreload 22737 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-09-29 07:02:18,250 autoreload 22750 8539431040 Watching for file changes with StatReloader
INFO 2023-10-01 01:32:30,264 autoreload 51283 8539431040 Watching for file changes with StatReloader
INFO 2023-10-01 01:32:35,575 views 51283 6130380800 Result: home page access!
ERROR 2023-10-01 01:32:35,621 log 51283 6130380800 Internal Server Error: /
Traceback (most recent call last):
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/myapp/views.py", line 15, in index
    return render(request, 'myapp/index.html', context=context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: myapp/index.html
ERROR 2023-10-01 01:32:35,680 basehttp 51283 6130380800 "GET / HTTP/1.1" 500 73850
WARNING 2023-10-01 01:32:35,951 log 51283 6147207168 Not Found: /apple-touch-icon-precomposed.png
WARNING 2023-10-01 01:32:35,952 basehttp 51283 6147207168 "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 3215
WARNING 2023-10-01 01:32:35,976 log 51283 6147207168 Not Found: /apple-touch-icon.png
WARNING 2023-10-01 01:32:35,977 basehttp 51283 6147207168 "GET /apple-touch-icon.png HTTP/1.1" 404 3179
WARNING 2023-10-01 01:32:56,455 log 51283 6130380800 Not Found: /myapp
WARNING 2023-10-01 01:32:56,455 basehttp 51283 6130380800 "GET /myapp HTTP/1.1" 404 3134
WARNING 2023-10-01 01:33:13,628 log 51283 6130380800 Not Found: /myapp/
WARNING 2023-10-01 01:33:13,628 basehttp 51283 6130380800 "GET /myapp/ HTTP/1.1" 404 3137
WARNING 2023-10-01 01:33:19,019 log 51283 6130380800 Not Found: /myapp/about
WARNING 2023-10-01 01:33:19,020 basehttp 51283 6130380800 "GET /myapp/about HTTP/1.1" 404 3152
WARNING 2023-10-01 01:33:25,663 log 51283 6130380800 Not Found: /myapp/about/
WARNING 2023-10-01 01:33:25,664 basehttp 51283 6130380800 "GET /myapp/about/ HTTP/1.1" 404 3155
WARNING 2023-10-01 01:33:36,197 log 51283 6130380800 Not Found: /myapp/eagle
WARNING 2023-10-01 01:33:36,198 basehttp 51283 6130380800 "GET /myapp/eagle HTTP/1.1" 404 3152
INFO 2023-10-01 01:33:43,240 views 51283 6130380800 Result: about page access
ERROR 2023-10-01 01:33:43,269 log 51283 6130380800 Internal Server Error: /about
Traceback (most recent call last):
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/myapp/views.py", line 23, in about
    return render(request, 'myapp/index.html', context=context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: myapp/index.html
ERROR 2023-10-01 01:33:43,271 basehttp 51283 6130380800 "GET /about HTTP/1.1" 500 73020
INFO 2023-10-01 01:35:06,669 autoreload 51476 8539431040 Watching for file changes with StatReloader
INFO 2023-10-01 01:35:09,151 views 51476 6165950464 Result: home page access!
ERROR 2023-10-01 01:35:09,175 log 51476 6165950464 Internal Server Error: /
Traceback (most recent call last):
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/myapp/views.py", line 15, in index
    return render(request, 'myapp/index.html', context=context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: myapp/index.html
ERROR 2023-10-01 01:35:09,176 basehttp 51476 6165950464 "GET / HTTP/1.1" 500 73850
INFO 2023-10-01 01:35:53,369 autoreload 51561 8539431040 Watching for file changes with StatReloader
INFO 2023-10-01 01:35:54,311 views 51561 6200045568 Result: home page access!
INFO 2023-10-01 01:35:54,314 basehttp 51561 6200045568 "GET / HTTP/1.1" 200 567
INFO 2023-10-01 01:36:02,563 views 51561 6200045568 Result: about page access
INFO 2023-10-01 01:36:02,564 basehttp 51561 6200045568 "GET /about HTTP/1.1" 200 568
INFO 2023-10-01 01:36:25,059 autoreload 51561 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp/views.py changed, reloading.
INFO 2023-10-01 01:36:25,418 autoreload 51602 8539431040 Watching for file changes with StatReloader
INFO 2023-10-01 01:36:31,644 views 51602 6121730048 Result: about page access
INFO 2023-10-01 01:36:31,649 basehttp 51602 6121730048 "GET /about HTTP/1.1" 200 373
INFO 2023-10-01 01:36:33,595 views 51602 6121730048 Result: about page access
INFO 2023-10-01 01:36:33,597 basehttp 51602 6121730048 "GET /about HTTP/1.1" 200 373
INFO 2023-10-01 01:37:31,780 views 51602 6121730048 Result: about page access
INFO 2023-10-01 01:37:31,783 basehttp 51602 6121730048 "GET /about HTTP/1.1" 200 373
INFO 2023-10-01 11:26:56,247 autoreload 54890 8539431040 Watching for file changes with StatReloader
INFO 2023-10-01 11:26:59,085 views 54890 6160658432 Result: home page access!
INFO 2023-10-01 11:26:59,092 basehttp 54890 6160658432 "GET / HTTP/1.1" 200 571
INFO 2023-10-01 11:27:08,673 views 54890 6160658432 Result: about page access
INFO 2023-10-01 11:27:08,682 basehttp 54890 6160658432 "GET /about HTTP/1.1" 200 573
INFO 2023-10-01 11:49:36,119 autoreload 54890 8539431040 /Users/aleksey/Desktop/Django/Seminar/games_app/views.py changed, reloading.
INFO 2023-10-01 11:49:36,557 autoreload 56370 8539431040 Watching for file changes with StatReloader
INFO 2023-10-01 11:50:09,714 autoreload 56427 8539431040 Watching for file changes with StatReloader
INFO 2023-10-01 11:50:45,871 autoreload 56476 8539431040 Watching for file changes with StatReloader
INFO 2023-10-01 11:51:24,166 autoreload 56534 8539431040 Watching for file changes with StatReloader
INFO 2023-10-01 11:51:27,008 views 56534 6165377024 Result: home page access!
INFO 2023-10-01 11:51:27,013 basehttp 56534 6165377024 "GET / HTTP/1.1" 200 578
INFO 2023-10-01 11:51:35,206 basehttp 56534 6165377024 "GET /eagle/6 HTTP/1.1" 200 673
INFO 2023-10-01 12:03:19,957 autoreload 56534 8539431040 /Users/aleksey/Desktop/Django/Seminar/myproject/urls.py changed, reloading.
INFO 2023-10-01 12:03:20,341 autoreload 57337 8539431040 Watching for file changes with StatReloader
INFO 2023-10-01 12:04:27,292 autoreload 57337 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/views.py changed, reloading.
INFO 2023-10-01 12:04:27,650 autoreload 57432 8539431040 Watching for file changes with StatReloader
INFO 2023-10-01 12:05:35,600 autoreload 57432 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/urls.py changed, reloading.
INFO 2023-10-01 12:05:35,947 autoreload 57523 8539431040 Watching for file changes with StatReloader
INFO 2023-10-01 12:05:51,217 autoreload 57523 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/urls.py changed, reloading.
INFO 2023-10-01 12:05:51,517 autoreload 57556 8539431040 Watching for file changes with StatReloader
INFO 2023-10-01 12:05:59,615 views 57556 6156054528 Result: home page access!
INFO 2023-10-01 12:05:59,624 basehttp 57556 6156054528 "GET / HTTP/1.1" 200 578
WARNING 2023-10-01 12:06:07,976 log 57556 6156054528 Not Found: /articles/2
WARNING 2023-10-01 12:06:07,977 basehttp 57556 6156054528 "GET /articles/2 HTTP/1.1" 404 3361
WARNING 2023-10-01 12:06:33,379 log 57556 6156054528 Not Found: /articles/2
WARNING 2023-10-01 12:06:33,380 basehttp 57556 6156054528 "GET /articles/2 HTTP/1.1" 404 3361
INFO 2023-10-01 12:07:05,077 autoreload 57556 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/urls.py changed, reloading.
INFO 2023-10-01 12:07:05,426 autoreload 57638 8539431040 Watching for file changes with StatReloader
INFO 2023-10-01 12:07:06,739 views 57638 6190116864 Result: home page access!
INFO 2023-10-01 12:07:06,743 basehttp 57638 6190116864 "GET / HTTP/1.1" 200 578
INFO 2023-10-01 12:07:12,361 basehttp 57638 6190116864 "GET /articles/2 HTTP/1.1" 200 0
INFO 2023-10-01 12:07:16,493 basehttp 57638 6190116864 "GET /articles/2 HTTP/1.1" 200 0
INFO 2023-10-01 12:14:56,026 autoreload 58133 8539431040 Watching for file changes with StatReloader
INFO 2023-10-01 12:14:57,831 views 58133 6134525952 Result: home page access!
INFO 2023-10-01 12:14:57,834 basehttp 58133 6134525952 "GET / HTTP/1.1" 200 578
INFO 2023-10-01 12:15:01,522 basehttp 58133 6134525952 "GET /articles/2 HTTP/1.1" 200 574
INFO 2023-10-01 12:15:09,381 basehttp 58133 6134525952 "GET /articles/2 HTTP/1.1" 200 574
INFO 2023-10-01 12:16:27,705 autoreload 58133 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/views.py changed, reloading.
INFO 2023-10-01 12:16:28,068 autoreload 58267 8539431040 Watching for file changes with StatReloader
INFO 2023-10-01 12:16:30,983 views 58267 6162608128 Result: home page access!
INFO 2023-10-01 12:16:30,988 basehttp 58267 6162608128 "GET / HTTP/1.1" 200 578
INFO 2023-10-01 12:16:34,459 basehttp 58267 6162608128 "GET /articles/2 HTTP/1.1" 200 574
INFO 2023-10-01 12:17:37,397 autoreload 58267 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/views.py changed, reloading.
INFO 2023-10-01 12:17:37,755 autoreload 58385 8539431040 Watching for file changes with StatReloader
INFO 2023-10-01 12:17:43,079 basehttp 58385 6188920832 "GET /articles/2 HTTP/1.1" 200 574
INFO 2023-10-01 12:17:58,287 autoreload 58385 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/views.py changed, reloading.
INFO 2023-10-01 12:17:58,611 autoreload 58420 8539431040 Watching for file changes with StatReloader
INFO 2023-10-01 12:18:05,836 basehttp 58420 6126972928 "GET /articles/2 HTTP/1.1" 200 574
INFO 2023-10-01 12:19:14,771 basehttp 58420 6126972928 "GET /articles/6 HTTP/1.1" 200 725
INFO 2023-10-01 12:19:39,320 autoreload 58420 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/views.py changed, reloading.
INFO 2023-10-01 12:19:39,712 autoreload 58595 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 02:11:43,371 autoreload 67058 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 02:11:45,431 views 67058 6133805056 Result: home page access!
INFO 2023-10-02 02:11:45,434 basehttp 67058 6133805056 "GET / HTTP/1.1" 200 578
ERROR 2023-10-02 02:11:52,808 log 67058 6133805056 Internal Server Error: /articles/6
Traceback (most recent call last):
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/myapp2/views.py", line 14, in get_articles
    return render(request, 'myapp/index.html', context=articles)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/backends/django.py", line 57, in render
    context = make_context(
              ^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/context.py", line 278, in make_context
    raise TypeError(
TypeError: context must be a dict rather than QuerySet.
ERROR 2023-10-02 02:11:52,809 basehttp 67058 6133805056 "GET /articles/6 HTTP/1.1" 500 89262
INFO 2023-10-02 02:12:24,203 autoreload 67134 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 02:12:25,628 views 67134 6171635712 Result: home page access!
INFO 2023-10-02 02:12:25,631 basehttp 67134 6171635712 "GET / HTTP/1.1" 200 578
ERROR 2023-10-02 02:12:29,367 log 67134 6171635712 Internal Server Error: /articles/6
Traceback (most recent call last):
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/myapp2/views.py", line 14, in get_articles
    return render(request, 'myapp/index.html', context=articles)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/backends/django.py", line 57, in render
    context = make_context(
              ^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/context.py", line 278, in make_context
    raise TypeError(
TypeError: context must be a dict rather than QuerySet.
ERROR 2023-10-02 02:12:29,368 basehttp 67134 6171635712 "GET /articles/6 HTTP/1.1" 500 89262
ERROR 2023-10-02 02:12:38,584 log 67134 6171635712 Internal Server Error: /articles/6
Traceback (most recent call last):
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/myapp2/views.py", line 14, in get_articles
    return render(request, 'myapp/index.html', context=articles)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/backends/django.py", line 57, in render
    context = make_context(
              ^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/context.py", line 278, in make_context
    raise TypeError(
TypeError: context must be a dict rather than QuerySet.
ERROR 2023-10-02 02:12:38,585 basehttp 67134 6171635712 "GET /articles/6 HTTP/1.1" 500 89262
INFO 2023-10-02 02:13:25,406 autoreload 67134 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/views.py changed, reloading.
INFO 2023-10-02 02:13:25,693 autoreload 67205 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 02:13:27,270 views 67205 6201077760 Result: home page access!
INFO 2023-10-02 02:13:27,272 basehttp 67205 6201077760 "GET / HTTP/1.1" 200 578
INFO 2023-10-02 02:13:30,029 basehttp 67205 6201077760 "GET /articles/6 HTTP/1.1" 200 725
INFO 2023-10-02 02:25:47,901 autoreload 68238 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 02:25:49,154 views 68238 6125727744 Result: home page access!
INFO 2023-10-02 02:25:49,157 basehttp 68238 6125727744 "GET / HTTP/1.1" 200 607
ERROR 2023-10-02 02:25:52,268 log 68238 6125727744 Internal Server Error: /articles/6
Traceback (most recent call last):
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/myapp2/views.py", line 14, in get_articles
    return render(request, 'myapp/index.html', context=context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/defaulttags.py", line 321, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/urls/base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/urls/resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'articles_full' not found. 'articles_full' is not a valid view function or pattern name.
ERROR 2023-10-02 02:25:52,274 basehttp 68238 6125727744 "GET /articles/6 HTTP/1.1" 500 241773
INFO 2023-10-02 02:26:40,668 autoreload 68319 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 02:26:41,706 views 68319 6165393408 Result: home page access!
INFO 2023-10-02 02:26:41,709 basehttp 68319 6165393408 "GET / HTTP/1.1" 200 607
INFO 2023-10-02 02:26:45,098 basehttp 68319 6165393408 "GET /articles/6 HTTP/1.1" 200 1796
INFO 2023-10-02 02:29:21,777 basehttp 68319 6165393408 "GET /articles/6 HTTP/1.1" 200 1520
INFO 2023-10-02 02:29:35,964 basehttp 68319 6165393408 "GET /articles/6 HTTP/1.1" 200 2171
INFO 2023-10-02 02:37:02,076 autoreload 68319 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/urls.py changed, reloading.
INFO 2023-10-02 02:37:02,362 autoreload 69123 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 02:37:36,993 autoreload 69123 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/views.py changed, reloading.
INFO 2023-10-02 02:37:37,335 autoreload 69189 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 02:38:58,514 autoreload 69189 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/views.py changed, reloading.
INFO 2023-10-02 02:38:58,801 autoreload 69315 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 02:39:26,335 autoreload 69315 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/views.py changed, reloading.
INFO 2023-10-02 02:39:26,602 autoreload 69357 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 02:42:02,055 views 69357 6170292224 Result: home page access!
INFO 2023-10-02 02:42:02,059 basehttp 69357 6170292224 "GET / HTTP/1.1" 200 607
INFO 2023-10-02 02:42:05,167 basehttp 69357 6170292224 "GET /articles/6 HTTP/1.1" 200 2203
INFO 2023-10-02 02:42:06,710 basehttp 69357 6170292224 "GET /detail/2 HTTP/1.1" 200 573
ERROR 2023-10-02 02:44:57,652 log 69357 6170292224 Internal Server Error: /detail/2
Traceback (most recent call last):
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/myapp2/views.py", line 22, in detail_articles
    return render(request, 'myapp/about.html', context=context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loader_tags.py", line 293, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loader_tags.py", line 232, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 484, in parse
    raise self.error(token, e)
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 482, in parse
    filter_expression = self.compile_filter(token.contents)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 600, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 703, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '{ description' from '{ description'
ERROR 2023-10-02 02:44:57,656 basehttp 69357 6170292224 "GET /detail/2 HTTP/1.1" 500 162446
ERROR 2023-10-02 02:45:04,346 log 69357 6170292224 Internal Server Error: /detail/3
Traceback (most recent call last):
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/myapp2/views.py", line 22, in detail_articles
    return render(request, 'myapp/about.html', context=context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loader_tags.py", line 293, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/loader_tags.py", line 232, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 484, in parse
    raise self.error(token, e)
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 482, in parse
    filter_expression = self.compile_filter(token.contents)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 600, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/template/base.py", line 703, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '{ description' from '{ description'
ERROR 2023-10-02 02:45:04,350 basehttp 69357 6170292224 "GET /detail/3 HTTP/1.1" 500 162446
INFO 2023-10-02 02:45:19,003 basehttp 69357 6170292224 "GET /detail/3 HTTP/1.1" 200 604
INFO 2023-10-02 02:45:26,008 basehttp 69357 6170292224 "GET /detail/3 HTTP/1.1" 200 604
INFO 2023-10-02 02:46:18,610 basehttp 69357 6170292224 "GET /detail/2 HTTP/1.1" 200 604
INFO 2023-10-02 02:46:27,469 basehttp 69357 6170292224 "GET /detail/2 HTTP/1.1" 200 604
INFO 2023-10-02 02:46:33,241 basehttp 69357 6170292224 "GET /detail/2 HTTP/1.1" 200 604
INFO 2023-10-02 02:46:57,280 autoreload 69357 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/views.py changed, reloading.
INFO 2023-10-02 02:46:57,566 autoreload 69940 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 02:46:59,922 basehttp 69940 6159446016 "GET /detail/2 HTTP/1.1" 200 604
INFO 2023-10-02 02:47:26,247 autoreload 69940 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/views.py changed, reloading.
INFO 2023-10-02 02:47:26,489 autoreload 69990 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 02:47:26,820 basehttp 69990 6203944960 "GET /detail/2 HTTP/1.1" 200 604
INFO 2023-10-02 02:47:30,708 basehttp 69990 6203944960 "GET /detail/2 HTTP/1.1" 200 604
INFO 2023-10-02 02:47:37,730 basehttp 69990 6203944960 "GET /detail/2 HTTP/1.1" 200 604
INFO 2023-10-02 02:47:42,457 basehttp 69990 6203944960 "GET /detail/4 HTTP/1.1" 200 604
INFO 2023-10-02 02:50:32,028 autoreload 69990 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/views.py changed, reloading.
INFO 2023-10-02 02:50:32,420 autoreload 70211 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 02:50:36,609 basehttp 70211 6156185600 "GET /articles/6 HTTP/1.1" 200 2203
INFO 2023-10-02 02:50:38,188 basehttp 70211 6156185600 "GET /detail/2 HTTP/1.1" 200 604
INFO 2023-10-02 02:52:21,687 autoreload 70211 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/views.py changed, reloading.
INFO 2023-10-02 02:52:21,955 autoreload 70385 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 02:52:24,489 basehttp 70385 6132002816 "GET /detail/2 HTTP/1.1" 200 610
INFO 2023-10-02 02:52:40,755 basehttp 70385 6132002816 "GET /detail/2 HTTP/1.1" 200 686
INFO 2023-10-02 02:53:54,766 autoreload 70385 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/views.py changed, reloading.
INFO 2023-10-02 02:53:55,046 autoreload 70551 8539431040 Watching for file changes with StatReloader
ERROR 2023-10-02 02:54:02,679 log 70551 6135377920 Internal Server Error: /detail/2
Traceback (most recent call last):
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/myapp2/views.py", line 19, in detail_articles
    articles.view += 1
    ^^^^^^^^^^^^^
AttributeError: 'Article' object has no attribute 'view'
ERROR 2023-10-02 02:54:02,680 basehttp 70551 6135377920 "GET /detail/2 HTTP/1.1" 500 59567
INFO 2023-10-02 02:56:31,221 autoreload 70551 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/views.py changed, reloading.
INFO 2023-10-02 02:56:31,500 autoreload 70789 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 02:56:31,898 basehttp 70789 6190067712 "GET /detail/2 HTTP/1.1" 200 686
INFO 2023-10-02 02:56:35,480 basehttp 70789 6190067712 "GET /detail/2 HTTP/1.1" 200 686
INFO 2023-10-02 03:00:06,861 autoreload 70789 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-10-02 03:00:07,257 autoreload 71079 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 03:00:42,936 autoreload 71079 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-10-02 03:00:43,270 autoreload 71136 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 03:00:47,464 autoreload 71136 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-10-02 03:00:47,788 autoreload 71161 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 03:01:47,649 autoreload 71161 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-10-02 03:01:48,082 autoreload 71263 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 03:02:11,653 autoreload 71263 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-10-02 03:02:11,985 autoreload 71319 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 03:02:49,790 autoreload 71319 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/models.py changed, reloading.
INFO 2023-10-02 03:02:50,215 autoreload 71386 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 03:19:24,421 autoreload 72710 8539431040 Watching for file changes with StatReloader
ERROR 2023-10-02 03:19:31,666 log 72710 6196801536 Internal Server Error: /detail/2
Traceback (most recent call last):
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/myapp2/views.py", line 19, in detail_articles
    comments = Comment.object.filter(article_id=article_id)
               ^^^^^^^^^^^^^^
AttributeError: type object 'Comment' has no attribute 'object'
ERROR 2023-10-02 03:19:31,666 basehttp 72710 6196801536 "GET /detail/2 HTTP/1.1" 500 59677
WARNING 2023-10-02 03:20:30,506 log 72710 6196801536 Not Found: /detail
WARNING 2023-10-02 03:20:30,507 basehttp 72710 6196801536 "GET /detail HTTP/1.1" 404 3548
ERROR 2023-10-02 03:20:37,054 log 72710 6196801536 Internal Server Error: /detail/2
Traceback (most recent call last):
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/myapp2/views.py", line 19, in detail_articles
    comments = Comment.object.filter(article_id=article_id)
               ^^^^^^^^^^^^^^
AttributeError: type object 'Comment' has no attribute 'object'
ERROR 2023-10-02 03:20:37,055 basehttp 72710 6196801536 "GET /detail/2 HTTP/1.1" 500 59677
INFO 2023-10-02 03:20:50,083 autoreload 72710 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/views.py changed, reloading.
INFO 2023-10-02 03:20:50,435 autoreload 72844 8539431040 Watching for file changes with StatReloader
ERROR 2023-10-02 03:20:54,041 log 72844 6193950720 Internal Server Error: /detail/2
Traceback (most recent call last):
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/myapp2/views.py", line 25, in detail_articles
    'comments': comments
                ^^^^^^^^
NameError: name 'comments' is not defined
ERROR 2023-10-02 03:20:54,042 basehttp 72844 6193950720 "GET /detail/2 HTTP/1.1" 500 58990
INFO 2023-10-02 03:21:07,938 autoreload 72844 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/views.py changed, reloading.
INFO 2023-10-02 03:21:08,253 autoreload 72891 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 03:21:08,877 basehttp 72891 6205108224 "GET /detail/2 HTTP/1.1" 200 710
INFO 2023-10-02 03:22:04,807 basehttp 72891 6205108224 "GET /detail/2 HTTP/1.1" 200 719
INFO 2023-10-02 03:22:31,675 autoreload 72891 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/views.py changed, reloading.
INFO 2023-10-02 03:22:32,052 autoreload 73000 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 03:25:38,527 autoreload 73000 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/views.py changed, reloading.
INFO 2023-10-02 03:25:38,909 autoreload 73290 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 03:26:03,449 autoreload 73290 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/views.py changed, reloading.
INFO 2023-10-02 03:26:03,778 autoreload 73327 8539431040 Watching for file changes with StatReloader
ERROR 2023-10-02 03:28:33,320 log 73327 6159085568 Internal Server Error: /detail/2
Traceback (most recent call last):
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/myapp2/views.py", line 19, in detail_articles
    comments = Comment.object.filter(article_id=article_id).order_by('-change_at')
               ^^^^^^^^^^^^^^
AttributeError: type object 'Comment' has no attribute 'object'
ERROR 2023-10-02 03:28:33,321 basehttp 73327 6159085568 "GET /detail/2 HTTP/1.1" 500 59743
INFO 2023-10-02 03:30:25,380 basehttp 73327 6159085568 "GET /articles/6 HTTP/1.1" 200 2203
ERROR 2023-10-02 03:30:36,388 log 73327 6159085568 Internal Server Error: /detail/2
Traceback (most recent call last):
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/myapp2/views.py", line 19, in detail_articles
    comments = Comment.object.filter(article_id=article_id).order_by('-change_at')
               ^^^^^^^^^^^^^^
AttributeError: type object 'Comment' has no attribute 'object'
ERROR 2023-10-02 03:30:36,389 basehttp 73327 6159085568 "GET /detail/2 HTTP/1.1" 500 59904
ERROR 2023-10-02 03:31:58,715 log 73327 6159085568 Internal Server Error: /detail/2
Traceback (most recent call last):
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/myapp2/views.py", line 19, in detail_articles
    comments = Comment.object.filter(article_id=article_id).order_by('-change_at')
               ^^^^^^^^^^^^^^
AttributeError: type object 'Comment' has no attribute 'object'
ERROR 2023-10-02 03:31:58,716 basehttp 73327 6159085568 "GET /detail/2 HTTP/1.1" 500 59743
INFO 2023-10-02 03:32:51,267 autoreload 73798 8539431040 Watching for file changes with StatReloader
ERROR 2023-10-02 03:32:54,850 log 73798 6196555776 Internal Server Error: /detail/2
Traceback (most recent call last):
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/myapp2/views.py", line 19, in detail_articles
    comments = Comment.object.filter(article_id=article_id).order_by('-change_at')
               ^^^^^^^^^^^^^^
AttributeError: type object 'Comment' has no attribute 'object'
ERROR 2023-10-02 03:32:54,851 basehttp 73798 6196555776 "GET /detail/2 HTTP/1.1" 500 59743
INFO 2023-10-02 03:33:14,790 autoreload 73798 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/views.py changed, reloading.
INFO 2023-10-02 03:33:15,173 autoreload 73832 8539431040 Watching for file changes with StatReloader
ERROR 2023-10-02 03:33:15,694 log 73832 6190477312 Internal Server Error: /detail/2
Traceback (most recent call last):
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aleksey/Desktop/Django/Seminar/myapp2/views.py", line 19, in detail_articles
    comments = Comment.object.filter()
               ^^^^^^^^^^^^^^
AttributeError: type object 'Comment' has no attribute 'object'
ERROR 2023-10-02 03:33:15,695 basehttp 73832 6190477312 "GET /detail/2 HTTP/1.1" 500 59635
INFO 2023-10-02 03:33:35,640 autoreload 73832 8539431040 /Users/aleksey/Desktop/Django/Seminar/myapp2/views.py changed, reloading.
INFO 2023-10-02 03:33:35,976 autoreload 73866 8539431040 Watching for file changes with StatReloader
INFO 2023-10-02 03:33:36,473 basehttp 73866 6169866240 "GET /detail/2 HTTP/1.1" 200 934
INFO 2023-10-02 03:34:24,422 basehttp 73866 6169866240 "GET /detail/2 HTTP/1.1" 200 920
INFO 2023-10-02 03:34:46,488 basehttp 73866 6169866240 "GET /detail/2 HTTP/1.1" 200 978
